### PR TITLE
[Frontend] Create dashboard page

### PR DESCRIPTION
**Pull request by brixtuyor**
# Related Links

- Task Link: http://127.0.0.1:8000/dashboard

# Description

- Refactored folder structure
- Changed components to pages
- Added components per page
- Created navbar

# Notes

- There are new npm packages added, please run `npm install`

# Updates

- Highlights the navigation link according to the current location
- Added light and dark mode

# Screenshots

image

### CODE REVIEW

File: resources/js/pages/dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Fragment } from "react";
+
+import Navbar from "./components/Navbar";
+
+export default function Dashboard() {
```
Comment (roseaugusto) [inline resources/js/pages/dashboard/Dashboard.tsx]: if directory is under the same name `dashboard` please call the file `index.js`

Comment (brixtuyor) [reply resources/js/pages/dashboard/Dashboard.tsx]: Changed the main folder files to `index.tsx`

File: resources/js/pages/dashboard/Activities.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Box } from "@chakra-ui/react";
+import Activity from "./Activity";
+
+export default function Activities() {
```
Comment (roseaugusto) [inline resources/js/pages/dashboard/Activities.tsx]: you can call this `ActivityList` so it is clearer and then for the Activity component you can change to `ActivityCard`

Comment (brixtuyor) [reply resources/js/pages/dashboard/Activities.tsx]: - Changed `Activities` to `ActivityList`
- Changed `Activity` to `ActivityCard`

**Review by roseaugusto — APPROVED** [review]
Approved!

**Pull request merged**